flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "end-of-life": "com.rtosta.zapzap",
```

Comment:
this doesn't do what you think it does. there is no reason to submit an app "end of life"

Reply:
> this doesn't do what you think it does. there is no reason to submit an app "end of life"

The idea is to do the Rebase, change of application ID.